Update "Basic operators, maths"

mahdiHashemi14 opened the pull request:
Translated some comments and strings
Changed some lines for better readability or translation
This article is done but not marked as checked in the [Translation Progress issue](https://github.com/javascript-tutorial/fa.javascript.info/issues/1)

mahdiHashemi14 commented:
Article of [Debugging in the browser](https://github.com/javascript-tutorial/fa.javascript.info/tree/master/1-js/03-code-quality/01-debugging-chrome) is also translated but not marked as checked in the issue

mahdyar reviewed (approved):
> Article of [Debugging in the browser](https://github.com/javascript-tutorial/fa.javascript.info/tree/master/1-js/03-code-quality/01-debugging-chrome) is also translated but not marked as checked in the issue

It's better to mention these types of issues at the issue itself (#1).
Its title had changed so @javascript-translate-bot couldn't recognize my PR (#93) and I wasn't a maintainer at the time. I updated it manually. :)

> Translated some comments and strings
> Changed some lines for better readability or translation
> This article is done but not marked as checked in the [Translation Progress issue](https://github.com/javascript-tutorial/fa.javascript.info/issues/1)

It was also translated in #19, and the title has changed since then. Updated that as well.

Thank you, @mahdiHashemi14!

mahdiHashemi14 commented:
> > Article of [Debugging in the browser](https://github.com/javascript-tutorial/fa.javascript.info/tree/master/1-js/03-code-quality/01-debugging-chrome) is also translated but not marked as checked in the issue
> 
> It's better to mention these types of issues at the issue itself (#1).
> Its title had changed so @javascript-translate-bot couldn't recognize my PR (#93) and I wasn't a maintainer at the time. I updated it manually. :)
> 
> > Translated some comments and strings
> > Changed some lines for better readability or translation
> > This article is done but not marked as checked in the [Translation Progress issue](https://github.com/javascript-tutorial/fa.javascript.info/issues/1)
> 
> It was also translated in #19, and the title has changed since then. Updated that as well.
> 
> Thank you, @mahdiHashemi14!

I always wondered why they weren't marked. It's like a relief for me to know :D
